modules/malltvAnalyticsAdapter.js

```diff
@@ -0,0 +1,190 @@
+import {ajax} from '../src/ajax.js'
+import adapter from '../src/AnalyticsAdapter.js'
+import CONSTANTS from '../src/constants.json'
+import adapterManager from '../src/adapterManager.js'
+import {getGlobal} from '../src/prebidGlobal.js'
+import {logInfo, logError, deepClone} from '../src/utils.js'
+
+const analyticsType = 'endpoint'
+export const ANALYTICS_VERSION = '1.0.0'
+export const DEFAULT_SERVER = 'https://central.mall.tv/analytics'
+
+const {
+  EVENTS: {
+    AUCTION_END,
+    BID_TIMEOUT
+  }
+} = CONSTANTS
+
+export const BIDDER_STATUS = {
+  BID: 1,
+  NO_BID: 2,
+  BID_WON: 3,
+  TIMEOUT: 4
+}
+
+export const getCpmInEur = function (bid) {
```

Comment:
@ardit-baloku 

Since this is now checking for getCpmInNewCurrency and the default is to return bid.cpm as is, do we need the initial condition? Seems like it can simply be 

```
export const getCpmInEur = function (bid) {
  if (typeof bid.getCpmInNewCurrency === 'function') {
    return bid.getCpmInNewCurrency('EUR')
  }

  return bid.cpm
}
```

Reply:
Doesn't seem necessary to call getCpmInNewCurrency if it is already in EUR

Reply:
Makes sense but this can still be modified just to remove the repeated return statement 

```
export const getCpmInEur = function (bid) {
  if (bid.currency !== 'EUR' && typeof bid.getCpmInNewCurrency === 'function') {
     return bid.getCpmInNewCurrency('EUR');
  }
  
  return bid.cpm;
```